### ISSUE TITLE
Use both replication and erasure coding

### ISSUE THREAD

**Issue by harveson** [sheepdog/sheepdog#397]
Is it possible to switch from replication to erasure coding without reformating a vdi?

**Comment by mitake** [comment]
No, it isn't supported. You need to export the VDI and import it again.
